DataDog/documentation
# read new `language_alias` param - [WEB-4412]

### What does this PR do? What is the motivation?

1. adjusts logic to read the new `language_alias` metadata from `static_analysis/rules`. this param is added to rules imported from an external repo. avoids the use of special characters (ie`#` in `C#`) in the url params.
   ```md
    
    meta:
      category: Security
      id: Loremipsum/urlopen-unsanitized-data
      language: C#
      language_alias: CSharp <---- NEW
      severity: Error
      severity_rank: 1
    ```
2. fixes pushState regex logic for deleting one url param so that the entire key:value is deleted.
   - deleting `Java` from selected languages would delete `Java` and a portion of `JavaScript`. Should only delete `Java`
3. updates `.ruleset-header__plus` count which indicates whether there are multiple languages within a ruleset over the language logo that is already displayed (i.e. `+1`,`+2` etc).
4. adds ci/cd synthetic tests for static analysis/rules page

jira: https://datadoghq.atlassian.net/jira/software/projects/WEB/boards/102?selectedIssue=WEB-4412

preview: https://docs-staging.datadoghq.com/stefon.simmons/read-language-alias-param/static_analysis/rules
   - to test:
      - verify filtering and search work with alias url param (i.e. `Csharp`)
      - `C#` should appear int he dropdown and the logo for `C#` should be `.NET`
      - url should filter rules on page load: https://docs-staging.datadoghq.com/stefon.simmons/read-language-alias-param/static_analysis/rules/?languages=CSharp
   
### Merge instructions


- [ ] Please merge after others review

### Additional notes
